packages/react/src/index.d.ts

```diff
@@ -3,1160 +3,1121 @@
+        disabled?: boolean;
+    }
+    export interface MultiStepsProps extends BasicProps {
+        current?: number;
+        currentLabel?: string;
+        onChange?: (current: number) => void;
+        direction?: 'horizontal' | 'vertical';
+        variant?:
+            | 'primary'
+            | 'accent'
+            | 'positive'
+            | 'warning'
+            | 'negative'
+            | 'white';
+    }
+    export const MultiSteps: React.FC<MultiStepsProps> & {
+        Item: React.FC<MultiStepsItemProps>;
+    };
+
+    interface OverlayBasicProps extends BasicProps {
+        children: React.ReactNode;
+        popperConfig?: PopperOptions;
+        placement?: PopperPlacement;
+        rootClose?: boolean;
+        rootCloseEvent?: string;
+        rootCloseDisabled?: boolean;
+    }
+    export interface OverlayTriggerProps extends OverlayBasicProps {
+        trigger?: TriggerType | TriggerType[];
+        delay?: number | { show: number; hide: number };
+        hoverOverlay?: boolean;
+        defaultShow?: boolean;
+        overlay: FuncType | React.ReactNode;
+    }
+
+    export interface OverlayProps extends OverlayBasicProps {
+        show?: boolean;
+        target?: React.RefObject<HTMLElement>;
+        container?: HTMLElement;
+        flip?: boolean;
+        containerPadding?: number;
+        onHide?: () => void;
+    }
+    export const Overlay: React.FC<OverlayProps> & {
+        Trigger: React.FC<OverlayTriggerProps>;
+    };
+
+    export interface PageLayoutProps extends BasicWithAsProps {
+        headerProps?: object;
+        footerProps?: object;
+        bodyProps?: object;
+    }
+    export const PageLayout: React.FC<PageLayoutProps> & {
+        Header: React.FC<BasicWithAsProps>;
+        Footer: React.FC<BasicWithAsProps>;
+        Body: React.FC<BasicWithAsProps>;
+    };
+
+    export interface PaginationItemProps extends BasicProps {
+        active?: boolean;
+        disabled?: boolean;
+    }
+    export interface PaginationProps extends BasicProps {
+        sizeControl?: InputSize;
+    }
+    export const Pagination: React.FC<PaginationProps> & {
+        Item: React.FC<PaginationItemProps>;
+    };
+
+    export interface ProblemInfoProps extends BasicProps {
+        topicLabel?: string;
+        topicName?: string;
+        descriptionLabel?: string;
+        descriptionValue?: string | FuncType;
+        additionalLabel?: string;
+        additionalValue?: string | FuncType;
+        src?: string;
+        onClickImage?: () => void;
+        action?: string | FuncType;
+    }
+    export const ProblemInfo: React.FC<ProblemInfoProps>;
+
+    export interface ProgressProps extends BasicAsProps {
+        variant?: 'primary' | 'accent' | 'positive' | 'warning' | 'negative';
+        now?: number;
+        height?: number;
+        label?: ReactNodeLike;
+        labelClassName?: string;
+        border?: boolean;
+        striped?: boolean;
+        animated?: boolean;
+    }
```

Comment:
The `label` prop can also receive a string value. Also, we should fix the default value of this prop in the component to be `''`

Reply:
The boolean fine, please see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/prop-types/index.d.ts#L22